packages/core/src/types.ts

```diff
@@ -740,6 +740,113 @@ export type DelayConfig<TContext, TEvent extends EventObject> =
   | number
   | DelayExpr<TContext, TEvent>;
 
+export type RequiredTypegenMachineOptions<
```

Comment:
Another nit: wondering if we should put this into a `types/metaTypes.ts` file or similar (the one huge file with all the types is my fault 🙈)